.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: node_js
 node_js:
     - stable
-
+script:
+  - node ./validation/markets-validator.js
```

Comment:
@torfsen What exactly do have to change to make let `npm test` execute the validation?